src/configs/all.ts

```diff
@@ -117,6 +117,7 @@ export const rules = {
     "no-use-before-declare": true,
     "no-var-keyword": true,
     "no-void-expression": true,
+    "prefer-conditional": true,
```

Comment:
can you please change the rule name? I think simply "conditional" is too vague because if-else statements are often described as conditionals as well. For example, [no-conditional-assignment](https://palantir.github.io/tslint/rules/no-conditional-assignment/) applies to the conditional expression as part of an `if` or `else` statement.

I would be happy with `prefer-conditional-operator` or `prefer-ternary-expression`